pkg/model/stack_serializer.go

```diff
@@ -0,0 +1,839 @@
+// Copyright 2020 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/okteto/okteto/pkg/log"
+	apiv1 "k8s.io/api/core/v1"
+)
+
+//Stack represents an okteto stack
+type StackRaw struct {
+	Version    string                 `yaml:"version,omitempty"`
+	Name       string                 `yaml:"name"`
+	Xname      string                 `yaml:"x-name,omitempty"`
```

Comment:
Let's remove the `x-....` fields altogether and wait for people asking for them to implement it. There might not be a clear user case for them